ui/shared/status/StatusStickersPopup.qml

```diff
@@ -148,27 +148,22 @@ Popup {
                     root.close()
                 }
             }
-            StatusStickerList {
-                id: loadingGrid
-                visible: chatsModel.stickers.recent.rowCount() === 0
-                interactive: false
-                model: new Array(20)
-                delegate: Item {
-                    width: stickerGrid.cellWidth
-                    height: stickerGrid.cellHeight
-                    Column {
-                        anchors.fill: parent
-                        anchors.topMargin: 4
-                        anchors.leftMargin: 4
-                        Rectangle {
-                            width: 80
-                            height: 80
-                            radius: width / 2
-                            color: Style.current.backgroundHover
-                        }
-                    }
+
+
+            Component {
+                id: loadingImageComponent
+                LoadingImage {
+                    width: 50
+                    height: 50
                 }
             }
+
+            Loader {
+                id: loadingGrid
+                active: chatsModel.stickers.recent.rowCount() === 0
```

Comment:
Wouldn't this condition mean that if I'm on a new install, the loading would never stop?

Reply:
good question.  Will check, but i guess that would happen with the original code too